tcms/rpc/tests/test_testrun.py

```diff
@@ -422,6 +454,10 @@ def verify_api_with_permission(self):
         self.assertIn("planned_stop", result)
         self.assertEqual(result["summary"], self.test_run_fields["summary"])
         self.assertIn("notes", result)
+        self.assertEqual(result["stop_date"], test_run.stop_date)
+        self.assertEqual(result["start_date"], test_run.start_date)
+        self.assertEqual(result["planned_start"], test_run.planned_start)
+        self.assertEqual(result["planned_stop"], test_run.planned_stop)
```

Comment:
If you are comparing the values directly with an assertion you don't need the `assertIn` statements for the same fields above because they are only checking whether or not the result dict contains keys with the given names. Comparing the values will blow up if they keys don't exist anyway and the test will fail. Which is what we want anyway if these fields become missing (become renamed, etc).